db/migrate/20180213204846_link_queue_and_task.rb

```diff
@@ -0,0 +1,5 @@
+class LinkQueueAndTask < ActiveRecord::Migration[5.0]
+  def change
+    add_reference :miq_queue, :miq_task, :type => "bigint", :index => true
```

Comment:
You shouldn't need the explicit `:type => "bigint"`, but I could be wrong.  Even so, we should probably use a sybol there instead of a string.  Also, not sure on adding an index.  @kbrock Thoughts on this index?